pybmix/core/mixing.py

```diff
@@ -314,3 +314,18 @@ def _check_args(self, n_comp, strength=None, discount=None, beta_params=None):
 
         else:
             raise ValueError("Not enough parameters provided")
+
+
+class PythonMixing(BaseMixing):
+    ID = mixing_id.PythonMix
+    NAME = mixing_id.MixingId.Name(ID)
+
+    def __init__(self, mix_implementation):
+        self._build_prior_proto()
+        self.mix_implementation = mix_implementation
```

Comment:
It gives a `ModuleNotFoundError: No module named 'something'`. It seemed to us quite self-explanatory error, do you think we need to add more logs?

Reply:
Yes I think it would be better to have a check raising a more detailed error. Everything is a "module" in python, I would just write "the mix_implementation file {0} does not exist".format(mix_implementation) or something along these lines

Reply:
We addressed this in the run_mcmc in the mixture_model.py since that's where the error actually raises. I just committed that, do you like this approach? The reason of using the old-fashioned style code was to avoid raising another ModuleNotFoundError and seeing the redundant traceback message like "During handling of the above exception, another exception occurred:" which would point to the same thing with a different message. 
If there is a neater solution, we'd like to know :)

Reply:
sounds good to me!